src/SWP/Component/ContentList/Model/ContentListItem.php

```diff
@@ -55,6 +55,7 @@ class ContentListItem implements ContentListItemInterface
     public function __construct()
     {
         $this->createdAt = new \DateTime();
+        $this->position = -1;
```

Comment:
hmm... good question - we we create article we want to add it to top of list (first place) or to end (last item)?